### PR TITLE
DE-6498: Add Storybook

### PR DESCRIPTION
**Pull request by fingerartur**
I Added storybook for API docs and development of the components.

Here I also did a few drive-by fixes of some bugs and the API, but more will come in [DE-6506](https://castlabs.atlassian.net/browse/DE-6506). There I will also fix the rest of the stories, bc. some of them currently do not work.

[DE-6506]: https://castlabs.atlassian.net/browse/DE-6506?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by fingerartur** [comment]
Okay, I fixed the Jest config, now it's really ready for review :)

File: story/stories/components/TimeLeft.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { Canvas, Meta, Primary, Controls, Story, ArgTypes } from '@storybook/blocks';
+import * as TimeLeftStories from './TimeLeft.stories';
+
+<Meta of={TimeLeftStories} name="Docs" />
+
+<Canvas of={TimeLeftStories.Primary} />
+
+# Time Left
+
+Time left.
```
Comment (doubeka) [inline story/stories/components/TimeLeft.mdx:11]:
```suggestion
A component to show the remaining playback duration.
```

File: story/stories/components/VolumeBar.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { Canvas, Meta, Primary, Controls, Story, ArgTypes } from '@storybook/blocks';
+import * as VolumeBarStories from './VolumeBar.stories';
+
+<Meta of={VolumeBarStories} name="Docs" />
+
+<Canvas of={VolumeBarStories.Primary} />
+
+# Volume Bar
+
+Volume bar.
```
Comment (doubeka) [inline story/stories/components/VolumeBar.mdx:11]:
```suggestion
A component to increase/decrease volume by dragging the slider.
```

**Review by doubeka — CHANGES_REQUESTED** [review]
This is such an great improvement of this documentation on these components, great job 💯 
I left few comments, mostly related to docs. 
Some additional extended descriptions for components in stories, when it makes (IMO) sense, some of the others may need another pass later and update even examples. (like spacer). 
I also checked the followup ticket, and I am happy to help and do some work on some following updates and improvements on various components APIs.  I would propose to schedule some planning session to flesh that plan furter.

**Comment by fingerartur** [comment]
Thanks!

**Review by doubeka — APPROVED** [review]

**Pull request merged**